.github/workflows/deploy-feature-envs.yaml

```diff
@@ -11,9 +11,9 @@ env:
   LABEL_DESTROYED: 'DESTROYED'
   LABEL_DEPLOYED: 'DEPLOYED'
   LABEL_DEPLOYED_CONTRACTS: 'DEPLOYED-CONTRACTS'
-  REGISTRY_HOST: 573243519133.dkr.ecr.us-east-1.amazonaws.com
-  FE_ALEPHNODE_REGISTRY: 573243519133.dkr.ecr.us-east-1.amazonaws.com/feature-env-aleph-node
-  FE_ALEPHNODE_REGISTRY_ESCAPED: '573243519133.dkr.ecr.us-east-1.amazonaws.com\/feature-env-aleph-node'
+  REGISTRY_HOST: public.ecr.aws
+  FE_ALEPHNODE_REGISTRY: public.ecr.aws/p6e8q1z1/feature-env-aleph-node
+  FE_ALEPHNODE_REGISTRY_ESCAPED: 'public.ecr.aws\/p6e8q1z1\/feature-env-aleph-node'
```

Comment:
what does this puppy do?

Reply:
So, it is used in one of the steps:
````
- name: Update feature environment to the latest PR image
        env:
          (...)
        run: |
          # Set up envs
          PR_IMAGE="${{ env.FE_ALEPHNODE_REGISTRY_ESCAPED }}\:${{ env.IMAGE_TAG }}"
          (...)
          sed "s/NODE_IMAGE/$PR_IMAGE/g" $ALEPH_PATH/aleph-apps/argocd/overlays/devnet/fe-apps/${{ env.APP_NAME }}.yaml > $ALEPH_PATH/aleph-apps/argocd/overlays/devnet/fe-apps/${{ env.APP_NAME }}.yaml.temp && mv $ALEPH_PATH/aleph-apps/argocd/overlays/devnet/fe-apps/${{ env.APP_NAME }}.yaml.temp $ALEPH_PATH/aleph-apps/argocd/overlays/devnet/fe-apps/${{ env.APP_NAME }}.yaml
````
For some reason, this variable needs to have slashes escaped.  It's used in `sed`.  I remember trying to remove that nasty thing with by taking the `FE_ALEPHNODE_REGISTRY` and just replacing `/` with `\/` but that was problematic.  Hence that ugly additional variable is added.